### PR TITLE
docs: Incorrect attributes in element doc

### PR DESCRIPTION
**Pull request by eliotwrobson**
See title. Fixes a typo.

### CODE REVIEW

File: docs/elements.md [hunk 1]
```diff
@@ -1906,7 +1906,7 @@ Displays boilerplate HTML from templates in a reusable way.
 #### Sample element
 
 ```html
-<pl-template file-name="outer_template.mustache" subdirectory="templates">
+<pl-template file-name="templates/outer_template.mustache" directory="serverFilesCourse">
```
Comment (jonatanschroeder) [inline docs/elements.md:1909]: `serverFilesCourse` is the default directory, maybe remove the attribute?

File: docs/elements.md [hunk 2]
```diff
@@ -1906,7 +1906,7 @@ Displays boilerplate HTML from templates in a reusable way.
...
   <pl-variable name="is-open">True</pl-variable>
   <pl-variable
```
Comment (jonatanschroeder) [inline docs/elements.md:1911]: (3 lines below, but I can't comment on GitHub): there is no such thing as `serverFilesQuestion`.

**Review by jonatanschroeder — COMMENTED** [review]
I know it's outside of the scope of the PR, but would it be an interesting idea to have a sample template file itself?

Also, lines 1939-1941, there is a comment about UUID, but in this particular context without a concrete example this can be more confusing than necessary.

File: docs/elements.md (hunk 1 above)
Comment (eliotwrobson) [reply docs/elements.md:1909]: Good catch! Going along with what you mentioned in your comment, I'll update this example with a template file that has a short example of using the uuid key.

File: docs/elements.md (hunk 2 above)
Comment (eliotwrobson) [reply docs/elements.md]: Done here https://github.com/PrairieLearn/PrairieLearn/pull/8434/commits/1bc546cef6db9bad95776c83a59fc83da54d36ad

File: docs/elements.md (hunk 1 above)
Comment (eliotwrobson) [reply docs/elements.md]: Done here https://github.com/PrairieLearn/PrairieLearn/pull/8434/commits/1bc546cef6db9bad95776c83a59fc83da54d36ad

File: docs/elements.md [hunk 3]
```diff
@@ -1901,21 +1901,47 @@ Printing Pandas DataFrames with this element is deprecated. Please use the new [
...
-  ></pl-variable>
+<pl-template file-name="templates/outer_template.mustache">
+  <pl-variable name="show">True</pl-variable>
+  <pl-variable name="section_header">This is the section header.</pl-variable>
+  <pl-variable name="section_body">This is the section body.</pl-variable>
 </pl-template>
 ```
 
+Along with the sample usage of the element, we include a sample template file. This is the file
+`templates/outer_template.mustache`:
```
Comment (jonatanschroeder) [inline docs/elements.md:1917]:
```suggestion
Along with the sample usage of the element, we include a sample template file. This is the file
`templates/outer_template.mustache`, stored in the course's `serverFilesCourse` directory:
```

File: docs/elements.md [hunk 4]
```diff
@@ -1901,21 +1901,47 @@ Printing Pandas DataFrames with this element is deprecated. Please use the new [
...
+
+```html
+<!-- Note that the id "#" CSS selector does not work for ids that start with a number, so must prefix uuids. -->
```
Comment (jonatanschroeder) [inline docs/elements.md:1920]: Since this comment is here, I believe the one in line 1967 is not needed.

**Review by jonatanschroeder — APPROVED** [review]

File: docs/elements.md (hunk 4 above)
Comment (eliotwrobson) [reply docs/elements.md]: Deleted this line here: https://github.com/PrairieLearn/PrairieLearn/pull/8434/commits/3acfec92b49ff72d8a2907691efef88e35b99bcc

**Pull request merged**